releases/release-1.31/release-notes/maps/pr-125176-map.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 125176
+releasenote:
+  text: |-
```

Comment:
It looks like this happened again: https://github.com/kubernetes/release/issues/3516 We can delete this file.